.github/workflows/main.yml

```diff
@@ -20,52 +22,25 @@ jobs:
       uses: microsoft/setup-msbuild@v1
 
     - name: Restore
-      run: msbuild /restore Xamarin.Forms.sln
-    
+      run: |
+        msbuild /restore Xamarin.Forms.sln
+        nuget restore Xamarin.Forms.Platform.GTK\Xamarin.Forms.Platform.GTK.csproj
+
     - name: Build
-      run: msbuild .\Xamarin.Forms.sln -p:Configuration=Release
+      run: |
+        msbuild .\Xamarin.Forms.sln -p:Configuration=${{env.CONFIGURATION}}
+        msbuild Xamarin.Forms.Platform.GTK\Xamarin.Forms.Platform.GTK.csproj /property:Configuration=${{env.CONFIGURATION}}
 
     - name: Package and Upload
       run: |
         git clone https://github.com/nblockchain/fsx
-        $env:DOTNETFORMS_VERSION = `.\fsx\Tools\fsi.bat fsx\Tools\nugetPush.fsx --output-version $env:BASE_VERSION`
+        $version = `.\fsx\Tools\fsi.bat fsx\Tools\nugetPush.fsx --output-version $env:BASE_VERSION`
 
-        echo ("DOTNETFORMS_VERSION=$env:DOTNETFORMS_VERSION") >> $env:GITHUB_ENV
-
-        nuget pack .nuspec/Xamarin.Forms.nuspec -p "version=$env:DOTNETFORMS_VERSION;Configuration=Release"
+        nuget pack .nuspec/Xamarin.Forms.nuspec -p "version=$version;Configuration=$env:CONFIGURATION"
+        nuget pack './.nuspec/Xamarin.Forms.Platform.GTK.nuspec' -p "version=$version;Configuration=$env:CONFIGURATION;core_version=$version"
```

Comment:
why do you use single-quotes here while in the first nuget pack command they were not used? let's be consistent